<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CapturaDialog</class>
 <widget class="QDialog" name="CapturaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelVideo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>531</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCapturar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>490</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
